### ISSUE TITLE
Zenbot buying, but not selling

### ISSUE THREAD

**Issue by githubee** [DeviaVir/zenbot#144]
I have been running Zenbot with near default configuration, but I only see it buying, not selling. Trend is always "null -> UP" - is that correct? Should there be an "UP -> DOWN" trend?

**Comment by nibzo1** [comment]
I'm seeing this too trading on Kraken with a default_logic.js which is a mix of the default one in the plugin directory and some changes found in the Issues threads. Using near defaults config. parameters

The Bot correctly decided on a buy position when the RSI hit 65+ but it never made any movement to sell on the downturn.

The price dropped substantially and the RSI was around 30. This is well below the buying price, and several hours into the price dip. I killed the Bot as it may have sold at a relatively large loss if the RSI hit 28 which is my lower limit.

I read in another thread that the bot should possibly be buying at this point as this is the bottom, and suggests a price correction, can anyone confirm this strategy? 

Does anyone know if the limit between trades effects both Buy and Sell.

So if the bot bought coins at X price will it have to wait until the trade time limit  passes until the bot can sell. Or does it only care about the limit between buys.

perhaps it is favorable to Hold for the trade time limit before commencing any trade activity?

**Comment by githubee** [comment]
I saw an RSI of 10 today, but bot didnt sell. It was 24+ hours after the last buy. I dont even see it trying or explaining why it didnt sell. strange.

Edit:
Tried to sell today. Maybe it was just the hype in the last days that prevented an earlier sell:

05/09/2017 10:56:01 AM CEST [    reporter]    m24905335       3 trades.        ---           RSI: 37 CLOSE:   0.04671000 BTC gdax.ETH-BTC BAL:   0.06467038 ROI:0.835 [link]
05/09/2017 10:56:01 AM CEST [      trader] gdax.ETH-BTC aborting sell due to low perf. = -0.165 [link]
05/09/2017 10:56:01 AM CEST [      trader] gdax.ETH-BTC attempting to sell 1.33072926 ETH for 0.06215836 BTC [link]
05/09/2017 10:56:01 AM CEST [      trader] gdax.ETH-BTC RSI:37 trend: null -> DOWN [link]
